src/components/common/__snapshots__/EdgeCreateDialog.test.js.snap

```diff
@@ -0,0 +1,239 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think that they need to be committed because they're used as comparison when the tests are run. With no comparison the tests would always pass. At least, this is how I interpret the process.

Reply:
Ok, interesting - I guess I would have expected that as part of the git CI, the tests would be run as part of the build process, and any snapshots would be generated then, not generated locally and then pushed to the repo?